echoregions/__init__.py

```diff
@@ -1,12 +1,21 @@
-from .convert.api import read_evl, read_evr
-from .convert.utils import parse_simrad_fname_time, parse_time
-from .formats import Lines, Regions2D
+from .lines.lines import Lines
+from .lines.lines_mask import lines_mask
+from .regions2d.regions2d import Regions2D
+from .regions2d.regions2d_mask import regions2d_mask
+from .utils.api import read_evl, read_evr
+from .utils.io import check_file, from_JSON, validate_path
+from .utils.time import parse_simrad_fname_time, parse_time
 
 __all__ = [
     "read_evl",
     "read_evr",
     "parse_simrad_fname_time",
     "parse_time",
+    "from_JSON",
+    "check_file",
+    "validate_path",
     "Lines",
     "Regions2D",
+    "lines_mask",
+    "regions2d_mask",
 ]
```

Comment:
I think the ones imported in this root level `__init__.py` are the one that users will use directly, such as doing 
```python
test_line = read_evl(SOME_EVL)
test_region = read_evr(SOME_EVR)
```
So I'd suggest to keep only the ones below:
```suggestion
__all__ = [
    "read_evl",
    "read_evr",
    "lines_mask",
    "regions2d_mask",
]
```
We probably should discuss again later (but soon) if we are happy with the way `lines_mask` and `regions2d_mask` are.